README.md

```diff
@@ -49,10 +49,12 @@ Key:
 * [Cucumber](https://cucumber.io/)
 * ~~[Cuke4Nuke](https://github.com/richardlawrence/Cuke4Nuke/wiki)~~
 * [Gauge](http://getgauge.io/)
+* [LightBDD](https://github.com/Suremaker/LightBDD)
 * **[Machine.Specifications](https://github.com/machine/machine.specifications)** (also known as MSpec)
 * ~~[NaturalSpec](https://github.com/forki/NaturalSpec)~~
 * [NBehave](http://nbehave.org/)
 * [NSpec](http://nspec.org/)
+* ~~[Spec4Net](https://bitbucket.org/fthomsen/spec4net/src)~~
```

Comment:
I don't know if Spec4Net is actually discontinued, but the last commit was on 2014-12-11.